ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/chiado.yaml

```diff
@@ -156,10 +156,15 @@ BELLATRIX_FORK_EPOCH: 180 # Mon Oct 10 2022 14:00:00 GMT+0000
 # Capella
 CAPELLA_FORK_VERSION: 0x0300006f
 CAPELLA_FORK_EPOCH: 244224 # Wed May 24 2023 13:12:00 GMT+0000
+# Deneb
+DENEB_FORK_VERSION: 0x0400006f
+DENEB_FORK_EPOCH: 516608 # Wed Jan 31 2024 18:15:40 GMT+0000
 
 INACTIVITY_SCORE_BIAS: 4
 # 2**4 (= 16)
 INACTIVITY_SCORE_RECOVERY_RATE: 16
+# [New in Deneb:EIP7514] 2**3 (= 8)
```

Comment:
2 in comment too (very nit)

Reply:
it was 8 in the comment in the gnosis :D

Reply:
`2*`
https://github.com/gnosischain/specs/blob/master/consensus/config/gnosis.yaml#L77

Reply:
bad edit, i'll just make ours 2